confidence/io.py

```diff
@@ -94,7 +91,7 @@ def dotted(name):
     return Configuration({dotted(name): value for name, value in values.items()})
 
 
-def read_envvar_file(name, extension):
+def read_envvar_file(name: str, extension: str = None) -> Configuration:
```

Comment:
Strictly speaking I think this should be type hinted as `extension: Optional[str] = None`